test/franklin_web/admin-user-stories/can_create_and_edit_with_article_editor_test.exs

```diff
@@ -0,0 +1,189 @@
+defmodule FranklinWeb.AdminUserStories.CanCreateAndEditWithArticleEditor do
+  @moduledoc """
+  Asserts the business rule that an authenticated admin can create or edit an
+  article via the article editor and that field value failures works as expected.
+  """
+
+  use FranklinWeb.ConnCase
+
+  alias Franklin.Articles
+  alias Franklin.Articles.Article
+
+  setup %{conn: conn} do
+    # Because the article editor is used for both the creation and editing of
+    # articles, we'll construct two instances of the live view, allowing us to
+    # shape the tests to verify aspects of the form for both its create version
```

Comment:
```suggestion
    # shape the tests to verify aspects of the form for both its create mode
```

I should be more consistent with editor terminology. Feel like I use the term `mode` in other parts of the code/comments.